### PR TITLE
Added second user github token. Made github token more dynamic. Added…

### PR DESCRIPTION
**Pull request by nagerz**
Added test/made pass for: 
-Edge case: Make sure this shows the proper repositories when there are more than one user in the database with different tokens.
-Don't display the "Github" section if the user is missing a Github token

### CODE REVIEW

File: app/services/github_service.rb [hunk 1]
```diff
@@ -11,7 +15,8 @@ def get_json(url)
 
   def conn
     Faraday.new(url: "https://api.github.com/") do |faraday|
-      faraday.headers["Authorization"] = "token #{ENV["GITHUB_TOKEN"]}"
+      #faraday.params["type"] = "owner"
```
Comment (nagerz) [inline app/services/github_service.rb:18]: We can discuss but we can add this line to limit repos to only the owners repos. @jmejia any thoughts as product owner?

**Review by teresa-m-knowles — APPROVED** [review]
Looks good. I like the option of sending in a user to the Github service, I think that works great. Also, I agree that we should restrict the repositories to those where the user is an owner, not just a collaborator or something.

File: app/facades/user_github_repos_facade.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 class UserGithubReposFacade
```
Comment (teresa-m-knowles) [inline app/facades/user_github_repos_facade.rb:1]: @nagerz Just realized we might have to change the name of this facade, since we might need other things for the dashboard that are not related to the github repos. I think facade is supposed to include everything that the view needs. So just a thought for the future.
